01_Core/week02/submissions/another-page.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <script type="text/javascript" src="javascript.js"></script>
+    </head>
+    <body>
+        <p id="dos">struggle bus with the last challenge</p>
+    </body>
+</html>
```

Comment:
This is super nit-picky, but all of your files should end with a blank line. If you are interested why, here's a quick link: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline